site/docs/guidelines/intercom.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: Intercom (Live chat)
+navTitle: Intercom (Live chat)
+summaryParagraph: Intercom (Live chat) is used to allow users to quickly access help or to send targeted messages to users.
+tags: ["Intercom", "Live chat"]
+needToKnow:
+  - It is a fixed element, and it it should appear in the same position across the product 
+  - It should not block anything on the page behind it.
+---
+
+import WhenToUseAndWhenNotToUse from "docs-components/WhenToUseAndWhenNotToUse"
+import WhenToUse from "docs-components/WhenToUse"
+import WhenNotToUse from "docs-components/WhenNotToUse"
+
+## Visuals
+
+<iframe style="border: none;" width="800" height="450" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2Fe3TyAOZKfLg9cA01Iy1Mgm%2FKaizen-Site-embed%3Fnode-id%3D53304%253A47849&chrome=DOCUMENTATION" allowfullscreen></iframe>
+
+## To keep in mind
+
+### When to use
+
+The Intercom icon can be used in these situations:
+
+- Allowing users to quickly access help (e.g. through contacting the support team or accessing the academy)
+- Sending targeted messages to users (e.g. to start conversations with users or to provide product updates)
+
+### Position
+
+- The Intercom icon should appear in the same position across the product (at the bottom right corner of the screen).
+- As it is a fixed element, it should not block anything on the page behind it.
+
+### Styling
+
+- The styling of the Intercom icon is done in the Intercom product 
+- Recommendations on how to style a custom Intercom icon:
+    - Your icon must be smaller than 50 kB.
+    - The optimum image size is 72px x 72px.
+    - A PNG or GIF with a transparent background works best, if using an image with a colored background, set the “Action color” in your Messenger settings to match. 
+
+
+## When to use and when not to use
+
+If in doubt, have a conversation with the Design Systems team or the design practice. If you feel strongly, talk about that.
```

Comment:
Given the absence of a titled design systems team, we can change this to "advocates" or remove the line here (we could consider removing it from the overview page too).